src/OSPSuite.UI/Controls/UxMasterDetailGridView.cs

```diff
@@ -0,0 +1,13 @@
+using DevExpress.XtraGrid;
+
+namespace OSPSuite.UI.Controls
+{
+   public class UxMasterDetailGridView : UxGridView
+   {
+      protected override string ViewName => "UxMasterDetailGridView";
```

Comment:
Not sure I understand why this subclass is required. I actually do not understand why we need the MasterXxxx . Can't we simply use UxGridView for all of that?
What exceptions are you referring to?